submissions/yhorodechna/side-menu-dom/js/script.js

```diff
@@ -0,0 +1,107 @@
+let _navEl;
+const getNavElement = () => {
+    if (!_navEl) {
+        _navEl = document.querySelector('#nav');
+    };
+    return _navEl;
+};
+let ACTIVE_EL_ID;
+
+function findHeaderLinkOrNavLink(clickedElement) {
```

Comment:
maybe better to make two separate listeners for header and navigation?

Reply:
Solved.